org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/task/TreeViewerDoubleClickListener.java

```diff
@@ -13,8 +13,11 @@
 
```

Comment:
The IP validation failed for your pull request:
<img width="721" alt="screen shot 2017-08-07 at 10 19 05" src="https://user-images.githubusercontent.com/419883/29018306-e84e4d8a-7b5a-11e7-8def-5d56a77b54d8.png">

The requirement from the Eclipse Foundation is to [have a `signed-off-by` footer](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#via_GitHub) in the commit messages.